dashboard/src/pages/TreeBuildDetails/TreeBuildDetails.tsx

```diff
@@ -90,7 +97,7 @@ const TreeBuildDetails = (): JSX.Element => {
       buildId={buildId}
       breadcrumb={breadcrumbElement}
       onClickFilter={onClickFilter}
-      tableFilter={searchParams.tableFilter}
+      tableFilter={searchParams.tableFilter ?? zTableFilterInfoDefault}
```

Comment:
is this needed? here you are using searchParams, not previousParams
also check this on the other files that have the same check, the other "...BuildDetails" files

Reply:
Even though this shouldn't be the case (since buildDetailsSearchSchema defines tableFilter and a default value to it) sometimes when changing the table filter this value can still be `undefined` and this was the bug we where facing where trying to change the table filter would cause the message error "something went wrong" to appear. For that reason, I added this verification